pkg/cmd/testdata/output_stdin_valid.txt

```diff
@@ -0,0 +1,18 @@
+# Ignoring 14 resources
+# Resources not covered by IaC
+aws_iam_user.driftctl
+aws_iam_user.sundowndev
+aws_iam_user.test_user
+aws_iam_role_policy.OrganizationAccountAccessRole:AdministratorAccess
+aws_iam_role_policy.driftctl_assume_role:driftctl_policy\.10
+aws_iam_role.OrganizationAccountAccessRole
+aws_iam_role.driftctl_assume\\_role
```

Comment:
This is unexpected, the original id in the input is `driftctl_assume\\_role` and thus the escaped ignored string should be `driftctl_assume\\\\_role`

Reply:
> This is unexpected, the original id in the input is driftctl_assume\\_role and thus the escaped ignored string should be driftctl_assume\\\\_role

The input is `driftctl_assume\\_role` but is serialized as `driftctl_assume\_role`, so the escaped ignored string is correct: `driftctl_assume\\_role`.

The JSON serializer doesn't accept `driftctl_assume\_role` because it means we try to escape `_` and produce the following error : `invalid character '_' in string escape code`

Reply:
🆗 👍🏻